.github/workflows/analysis-coverage.yml

```
@@ -338,7 +338,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        nextcloud: [ 'stable31', 'stable32' ]
+        nextcloud: [ 'stable32', 'stable34' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)